### PR TITLE
Add HyperTerm

### PR DESCRIPTION
**Pull request by bnb**
**https://github.com/bnb/awesome-hyperterm**

`awesome-hyperterm` is a list of awesome plugins, themes, and resources for the Hyperterm app, which is an electron-based terminal that's insanely extensible and (subjective) just really damn awesome (/subjective).

# ALL THE BELOW CHECKBOXES SHOULD BE DONE AND CHECKED AND THE ABOVE PLACEHOLDERS SHOULD BE FILLED IN BEFORE SUBMITTING A PULL REQUEST!

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. *(For example: `Add Node.js`)*
- The list I added
	- [x] **has been around for at least 20 days,**
	- [x] is a non-generated Markdown file in a GitHub repo,
	- [x] is not a duplicate,
	- [x] only has awesome items with descriptions *(Awesome lists are curations of the best, not of everything)*,
	- [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- [x] has a Table of Contents section named `Contents` as the first section,
	- [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)*,
	- [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- [x] has been checked for inconsistent formatting, spelling, and grammar *(This includes starting each link description with an uppercase character and ending it with a period `.`, and consistent naming)*,
	- [x] has been added at the bottom of the appropriate category.

## Commit Comment
Adding the `awesome-hyperterm` (https://github.com/bnb/awesome-hyperterm) list to the `Development Environment` section of Awesome lists!

### CODE REVIEW

**Comment by sindresorhus** [comment]
Generally looks great. Just some nitpick:

- Section header should be `##`, not `#`. `#` is for the heading.
- `Github` => `GitHub`
- `Zsh for HyperTerm` => `ZSH for HyperTerm.`
- `Hyperterm` => `HyperTerm`.

File: readme.md [hunk 1]
```diff
@@ -299,7 +299,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Git Add-ons](https://github.com/stevemao/awesome-git-addons)
 - [SSH](https://github.com/moul/awesome-ssh)
 - [FOSS for Developers](https://github.com/httpsGithubParty/FOSS-for-Dev)
-
```
Comment (sindresorhus) [inline readme.md]: Don't remove the newline.

File: readme.md [hunk 2]
```diff
@@ -299,7 +299,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
...
+- [Hyperterm](https://github.com/bnb/awesome-hyperterm)
```
Comment (sindresorhus) [inline readme.md]: Hyperterm => HyperTerm

**Comment by sindresorhus** [comment]
ping @bnb

**Comment by bnb** [comment]
@sindresorhus Sorry, got a new Mac and haven't had GitHub notifications.

**Comment by bnb** [comment]
@sindresorhus Updated!

**Comment by bnb** [comment]
And I accidentially hit command + enter on that last commit via GitHub Web UI, so the commit name got f'ed up. Was supposed to be `-> "T"` 🙄

File: readme.md [hunk 3]
```diff
@@ -299,7 +299,8 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Git Add-ons](https://github.com/stevemao/awesome-git-addons)
 - [SSH](https://github.com/moul/awesome-ssh)
 - [FOSS for Developers](https://github.com/httpsGithubParty/FOSS-for-Dev)
-
+- [HyperTerm](https://github.com/bnb/awesome-hyperterm)
+-
```
Comment (sindresorhus) [inline readme.md]: Uhm, you're now adding an `-`.

Comment (sindresorhus) [inline readme.md]: Will fix during merge.

**Comment by bnb** [comment]
I'm dumb. 🙄 

Anyway, thanks for the merge! 🤗

**Comment by sindresorhus** [comment]
No worries :)

**Pull request merged**